lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -1178,6 +1178,8 @@ bool lldb_private::formatters::swift::SIMDVector_SummaryProvider(
       return false;
 
     auto synthetic = simd_elem->GetSyntheticValue();
+    if (!simd_elem)
```

Comment:
This is after the access? Or is this supposed to check `synthetic`?

Reply:
Good thing we do code reviews!